azure-pipelines.yml

```diff
@@ -36,8 +40,18 @@ jobs:
     condition: contains(variables['Agent.JobName'], 'windows')
   - script: npm ci
     displayName: Install dependencies
-  - script: node js2bin.js --ci --node=14.15.3 --size=2MB --size=4MB --upload --clean
-    displayName: Build base node binaries
+  - script: docker run --rm --privileged multiarch/qemu-user-static --reset -p yes
+    displayName: setup docker for multi platform
+    condition: contains(variables['Agent.JobName'], 'linux')
+  - script: node js2bin.js --ci --container=true --node=14.15.1 --size=2MB --size=4MB --upload --clean --arch=${ARCH}
+    displayName: Build base node binaries non-X64
+    condition: contains(variables['Agent.JobName'], 'linux')
+    env:
+      GITHUB_TOKEN: $(PersonalGithubToken)
+      ARCH: $(arch)
+  - script: node js2bin.js --ci --container=true --node=14.15.1 --size=2MB --size=4MB --upload --clean
```

Comment:
these steps are never going to be executed - based on the condition

Reply:
They do get executed on Mac & Windows.